WebRoot/js/zimbraMail/briefcase/model/ZmBriefcaseItem.js

```diff
@@ -359,9 +359,16 @@ function() {
 
 ZmBriefcaseBaseItem.prototype._loadFromDom =
 function(node) {
-
-	this.id = node.id;
-
+    /* In search api response we get 'id' and 'sfid' for files in 'files shared with me' folder. Here id is a combination id which is required to form the rest url and get the prview data and sfid is the actual 'id' of the item/file.
```

Comment:
indentation
also please match comment styling as per standard

Reply:
fixed indentation and comment syling.